core/interfaces.go

```diff
@@ -97,7 +97,7 @@ type CertificateAuthority interface {
 
 // PolicyAuthority defines the public interface for the Boulder PA
 type PolicyAuthority interface {
-	WillingToIssue(AcmeIdentifier) error
+	WillingToIssue(id AcmeIdentifier, regID int64) error
```

Comment:
nit: So I totally think *this* is the right way to do this, but the current convention for this file is to just use the argument types.